tree/tree/src/TBranch.cxx

```diff
@@ -1279,6 +1279,14 @@ Int_t TBranch::GetEntry(Long64_t entry, Int_t getall)
             fNextBasketEntry = -1;
             return -1;
          }
+         if (fTree->GetMaxVirtualSize() < 0) {
```

Comment:
Hi Philippe, Brian,
the intention was to make GetMaxVirtualSize similar to TTreeCache size, where a positive value indicates the number of bytes and a negative value uses clusters. 
Even though one could separate the pre-loading of trailing cluster baskets ('load-whole-cluster-at-once') and the retaining of clusters, I think they are very correlated pre-loading clusters probably only makes sense if one also drops clusters, rather than baskets. Maybe I don't understand this correctly, but what is your suggestion?